cli/types/cypress.d.ts

```diff
@@ -657,6 +657,20 @@ declare namespace Cypress {
      */
     as(alias: string): Chainable<Subject>
 
+    /**
+     * Attach a file to the given <input> element, or drag and drop a file over any subject.
+     *
+     * @param {FileReference} files - The file(s) to attach or drag onto this element.
+     * @see https://on.cypress.io/selectFile
```

Comment:
Has this on link been created on Services? I think this is also meant to be all lowercaps to reroute correctly. 
```suggestion
     * @see https://on.cypress.io/selectfile
```

Reply:
It hasn't been added yet - I'll open a PR to do that, thanks for the reminder.

I'm lowercasing the link for consistency, but worth noting that URLs are almost universally case insensitive on all modern systems. This isn't given in any spec, and it's not *universally* true, but in most cases it is.

https://docs.cypReSS.io/api/cOmMaNdS/wItHiN

(let's never actually do this, just an interesting note)

Reply:
https://github.com/cypress-io/cypress-services/pull/4174